db/upgrade.php

```diff
@@ -164,5 +164,28 @@ function xmldb_accredible_upgrade($oldversion=0) {
         upgrade_mod_savepoint(true, 2022060900, 'accredible');
     }
 
+    if ($oldversion < 2024041600) {
+        // Define field finalgradetopass to be added to accredible.
+        $table = new xmldb_table('accredible');
+        $field = new xmldb_field('finalgradetopass', XMLDB_TYPE_INTEGER, '3', null, XMLDB_NOTNULL, null, '0', 'groupid');
+
+        // Conditionally launch add field finalgradetopass.
+        if (!$dbman->field_exists($table, $field)) {
+            $dbman->add_field($table, $field);
+        }
+
+        // Define field attributemapping to be added to accredible.
+        $table = new xmldb_table('accredible');
```

Comment:
You don't need this here, it's already defined in line 169

Reply:
You're right! Thanks.